### PR TITLE
Enable multiple editors - Plone 5.1

### PR DESCRIPTION
**Pull request by gotcha**
Depends on https://github.com/plone/plone.app.textfield/pull/44 (merged by @mauritsvanrees)

~~Antoine Duchêne contributor agreement is being prepared.~~ Antoine's contributor agreement is now in.

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 package-extras = [tests]
+auto-checkout = plone.app.textfield
+sources-dir = devel
 
 [versions]
 plone.app.z3cform =
+zc.buildout = 3.0.0b3
+
+[sources]
+plone.app.textfield = git git@github.com:gotcha/plone.app.textfield.git branch=ckeditor_1.2
```
Comment (mauritsvanrees) [inline buildout.cfg:14]: I have released `plone.app.textfield` 1.2.12, so you can use that.

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
LGTM, though I have not tried it.

One inline comment about the source/version to use.

File: buildout.cfg (hunk 1 above)
Comment (gotcha) [reply buildout.cfg]: See new commits

**Comment by gotcha** [comment]
Change included. Thx for the review.

@mauritsvanrees Would you make a release or enable me to do it ?

**Comment by mauritsvanrees** [comment]
I have released this in 3.0.11.
You already have PyPI rights.

**Pull request merged**